components/member_list_channel/member_list_channel.jsx

```diff
@@ -144,14 +145,18 @@ export default class MemberListChannel extends React.Component {
                 const user = users[i];
 
                 if (teamMembers[user.id] && channelMembers[user.id] && user.delete_at === 0) {
-                    usersToDisplay.push(user);
+                    const status = UserStore.getStatus(user.id);
```

Comment:
We shouldn't be using anything not in props or state to render. We should pull the statuses to when we set the users in state or we should rewrite this component to be more redux friendly and pass it in as props